ynput/ayon-resolve
# Resolve: publishing editorial pkg from active timeline

We've upgraded the loader in several ways:

*   Collect all instances of timelines labeled as "editorial package" from the mediapool. Those could be loaded timelines via dedicated loader for editorial package or created via Creator.


This PR is based on #11

closes #25 
closes #19 

## Testing steps
- select any timeline with clips and use Create with `Editorial Package` product type
- context selector window should popup 
- select folder and task context which should be used for the product
- close Creator and open Publisher
- publish and all should look as expected - editorial pkg product is now ready to be loaded.
